packages/instant/src/components/buy_button.tsx

```diff
@@ -89,12 +95,14 @@ export class BuyButton extends React.Component<BuyButtonProps> {
         try {
```

Comment:
We could do the `submitted` event here and report the expected time in ms -- this might be useful to report